cluster/manager_delete.go

```diff
@@ -30,6 +31,35 @@ func (m *Manager) DeleteCluster(ctx context.Context, cluster CommonCluster, forc
 	return nil
 }
 
+func deleteAllResource(kubeConfig []byte, logger *logrus.Entry) error {
+	client, err := helm.GetK8sConnection(kubeConfig)
+	if err != nil {
+		return err
+	}
+	// Delete all resources, log errors but ignore them
+	err = client.CoreV1().Services("").Delete("", metav1.NewDeleteOptions(0))
+	if err != nil {
+		logger.Info(err.Error())
```

Comment:
Returning a wrapped error would be better here, so that the caller can log the error if it wants to.